scripts/createUmlsRelations.py

```diff
@@ -0,0 +1,29 @@
+"""
+"""
+
+from collections import defaultdict
+from sys import argv
+
+def getRelations(mrrel):
+	# PAR ~ parent
+	# RB  ~ broader
+	eligibleRelations = set(["PAR", "RB"])
+	for line in open(mrrel,"r"):
+		line    = line.split(r"|")
+		cui1    = line[0]
+		idType1 = line[2]
+		cui2    = line[4]
+		idType2 = line[6]
+		rel     = line[3]
+		suppressed = line[14]
+		# Filter for CUI id references because those are the concept relations.
+		if rel in eligibleRelations and suppressed == "N" and idType1 == "CUI" and idType2 == "CUI":
+			yield "\t".join([cui1, rel, cui2])
+
+def main():
+	mrrel = argv[1]
```

Comment:
What should be argv[1] here? The `MRREL.RRF` file?

Reply:
Yes. Yeah, sorry, I did forget to add a call example to the file.